services/galley/src/Galley/API/MLS/Message.hs

```diff
@@ -475,7 +476,15 @@ postMLSMessageToLocalConv ::
   Sem r [LocalConversationUpdate]
 postMLSMessageToLocalConv qusr senderClient con smsg lcnv = case rmValue smsg of
   SomeMessage tag msg -> do
-    conv <- getLocalConvForUser qusr lcnv
+    gtc <- getGlobalTeamConversationById lcnv
+    conv <- case gtc of
+      Just conv -> do
+        when (isNothing (gtcCreator conv)) $ do
+          setGlobalTeamConversationCreator conv (qUnqualified qusr)
+        localMembers <- getLocalMembers (qUnqualified . gtcId $ conv)
```

Comment:
Won't this return the empty list?

Reply:
It now returns the same results as they member list already in the conversation. Dead code.